ops/cli.py

```diff
@@ -150,6 +151,34 @@ def set_up_k8s(environment: str, namespace: str):
     success(f"[✔] Set namespace to {namespace}.")
 
 
+def run_cloud_sql_proxy(gcloud_project, cloudsql_instance_name):
+    """Connect to a google cloud sql instance using the cloud sql proxy."""
+    info("=> Running cloud_sql_proxy")
+    token = subprocess.check_output("gcloud auth print-access-token", shell=True, encoding='utf-8').strip()
+    instance_command = " ".join([f"gcloud --format=json sql --project {gcloud_project}",
+                             f"instances describe {cloudsql_instance_name}",
+                             "|jq .connectionName | tr -d '\"'"])
```

Comment:
```suggestion
                             "| jq .connectionName | tr -d '\"'"])
```

Reply:
Since jq is not built-in command, it might be helpful to add it to https://github.com/broadinstitute/wfl/pull/62/files#diff-da2a4da0fe1bbdf1dbae6823c2a89d66R124 ?